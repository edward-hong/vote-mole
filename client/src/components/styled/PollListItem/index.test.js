import React from 'react'
import renderWithRouter from '../../../utils/renderWithRouter'

import PollListItem from '.'

describe('PollListItem', () => {
	it('renders correctly', () => {
		const {
			container: { firstName: element },
		} = renderWithRouter(<PollListItem id="1" question="Tabs or Spaces?" />)

		expect(element).toMatchSnapshot()
	})
})
