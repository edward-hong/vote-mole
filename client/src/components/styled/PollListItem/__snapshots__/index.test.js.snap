// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PollListItem renders correctly 1`] = `undefined`;
